### PR TITLE
gpg: add pt_BR translation

### PR DESCRIPTION
**Pull request by caduvieira**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages.pt_BR/common/gpg.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Descriptografa doc.txt.gpg (envia saída para `stdout`):
+
+`gpg --decrypt {{doc.txt.gpg}}`
+
+- Importa uma chave pública:
+
+`gpg --import {{public.gpg}}`
+
+- Exporta  a chave púbica da alice@example.com (envia saída para `stdout`):
```
Comment (andrealmeid) [inline pages.pt_BR/common/gpg.md:26]:
```suggestion
- Exporta a chave púbica da alice@example.com (envia saída para `stdout`):
```

File: pages.pt_BR/common/gpg.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# gpg
+
+> GNU Privacy Guard.
+> Mais informações: <https://gnupg.org>.
+
+- Assinar doc.txt sem criptografia (cria um arquivo de saída doc.txt.asc):
```
Comment (andrealmeid) [inline pages.pt_BR/common/gpg.md:6]: All other verbs are in imperative form, I think this one should be as well

```suggestion
- Assina doc.txt, sem criptografá-lo (cria um arquivo de saída doc.txt.asc):
```

File: pages.pt_BR/common/gpg.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Assina doc.txt, sem criptografá-lo (cria um arquivo de saída doc.txt.asc):
+
+`gpg --clearsign {{doc.txt}}`
+
+- Criptografa doc.txt para alice@example.com (cria um arquivo de saída doc.txt.gpg):
+
+`gpg --encrypt --recipient {{alice@example.com}} {{doc.txt}}`
+
+- Criptografa doc.txt com uma senha simétrica somente (cria um arquivo de sadída doc.txt.gpg):
```
Comment (andrealmeid) [inline pages.pt_BR/common/gpg.md:14]: @caduvieira: what do you think about that?

```suggestion
- Criptografa doc.txt apenas com uma senha simétrica (cria um arquivo de sadída doc.txt.gpg):
```

Comment (caduvieira) [reply pages.pt_BR/common/gpg.md]: I think we can use your suggestion.

File: pages.pt_BR/common/gpg.md (hunk 1 above)
Comment (caduvieira) [reply pages.pt_BR/common/gpg.md]: I changed it. There is a typo also in "púbica". It should be "pública".

File: pages.pt_BR/common/gpg.md (hunk 3 above)
Comment (andrealmeid) [reply pages.pt_BR/common/gpg.md]: cool :)

**Comment by andrealmeid** [comment]
@navarroaxel I can't do an official approval, but I endorse yours, from a native pt_BR speaker :)

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

@andrealmeid: If you have 5 non-trivial PRs merged, according to our [community roles](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md) document you'd be eligible to become a contributor :D

**Comment by andrealmeid** [comment]
> Don't speak the language, but the syntax looks fine to me!
> 
> Thanks slightly_smiling_face
> 
> @andrealmeid: If you have 5 non-trivial PRs merged, according to our [community roles](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md) document you'd be eligible to become a contributor :D

@sbrl: cool! I'll have a closer look at this later :)

**Pull request merged**